sys/include/net/gnrc/rpl.h

```diff
@@ -32,11 +32,12 @@
  * Auto-Initialization
  * -------------------
  *
- * If the application defines only one interface (`GNRC_NETIF_NUMOF == 1`),
- * then RPL will be initialized on this interface.
+ * If the application defines only one interface (@ref gnrc_netif_highlander()
```

Comment:
Shouldn't this have been part of #13226 :-/?

Reply:
I didn't remove GNRC_NETIF_NUMOF there, so there's a gray zone between this and #13226